package.json

```diff
@@ -4,11 +4,11 @@
   "description": "elastic.io component that connects to Sugar CRM API (node.js)",
   "main": "index.js",
   "engines": {
-    "node": "8.1.0"
+    "node": "10.x.x"
```

Comment:
`elasticio-sailor-nodejs` >= 2.6.0 requires `node` = 12+

Reply:
Updated to 12.x.x